typespec-extension/src/code-model-builder.ts

```diff
@@ -2229,35 +2223,39 @@ export class CodeModelBuilder {
 
   private processMultipartFormDataFilePropertySchemaFromSdkType(
```

Comment:
So this function is not only for `FileProperty`? Do we want to rename the function to `processMultipartFormDataSchemaFromSdkType`?

Reply:
Currently it is only for File property
https://github.com/Azure/autorest.java/blob/6f322419cb50c6a342855ab70617bce96365e6cf/typespec-extension/src/code-model-builder.ts#L2104-L2106

Reply:
I'm a bit confused. Then why `getFileDetailsSchema` function can have 'model' type, I thought files are all bytes.

https://github.com/Azure/autorest.java/blob/6f322419cb50c6a342855ab70617bce96365e6cf/typespec-extension/src/external-schemas.ts#L209

Reply:
`HttpPart<File>`

that `File` is model

Reply:
I see, Thanks